docs/source/technical_tutorials/search/cmr_search_in_r.ipynb

```diff
@@ -0,0 +1,358 @@
+{
```

Comment:
<p>Should we show how <em>paginate </em>results are narrow down by some other filter? The following spatial one is great, but maybe we could show an example of filtering based on some sort of key? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/456/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>To do: find granules with a better selection of keys</p>